docpipeline/generate.py

```diff
@@ -209,6 +196,10 @@ def build_and_format(blob, is_bucket, devsite_template):
         hide_output=False,
     )
 
+    # make final adjustments to java toc
+    if metadata.language.lower() == "java":
+        prepare.prepare_java_toc(site_path.joinpath("toc.yaml"), metadata.name)
```

Comment:
Please move this down (or the `add_prettyprint` call up) so that they're right next to each other. Might be better to move down so we can use the `_toc.yaml` name.